persist/pom.xml

```diff
@@ -14,7 +12,7 @@
   <properties>
     <flyway.version>6.0.3</flyway.version>
     <database.driver>org.postgresql.Driver</database.driver>
-    <database.url>jdbc:postgresql://localhost:5434/checkin?autoreconnect=true</database.url>
+    <database.url>jdbc:postgresql://localhost:5432/c4cneu-db?autoreconnect=true</database.url>
```

Comment:
There's a commit on the backend-scaffold that pulls the database params into a config file so that you don't have to specify them in multiple places, consider bringing that commit in